tests/phpunit/BltProject/ToggleModulesTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Acquia\Blt\Tests\BltProject;
+
+use Acquia\Blt\Tests\BltProjectTestBase;
+
+/**
+ * Class ToggleModulesTest.
+ *
+ * Verifies that setup:toggle-modules behaves as expected.
+ */
+class ToggleModulesTest extends BltProjectTestBase {
+
+  /**
+   * Verifies the modules for a given environment were enabled as expected.
+   *
+   * In the event no environment is specified, this test will be skipped.
+   *
+   * @group blt-project
+   */
+  public function testModulesEnabled() {
+    if (defined('BLT_ENV')) {
```

Comment:
Yeah, I agree with you there. I just didn't want to cause unexpected results here since this is the only test invokation that has the `phpunit.xml` being properly passed in at this point. But if we can assume that all phpunit tests will have access to the configuration I see no problem with removing the conditionals. 

Perhaps an issue should be opened to ensure that the travis scripts have this configured?